reports/summary-2020.md

```diff
@@ -0,0 +1,844 @@
+away from complications with Colon Cancer. Dan was instrumental in shaping both
+the Kubernetes project, the CNCF, and the Cloud Native community as a whole. He
+understood that a foundation built on a vibrant and diverse community was a
+requirement to be successful, and the project would not be what it is today
+without him. 
+
+[November of 2020, Dan Kohn]: https://kubernetes.io/blog/2020/11/02/remembering-dan-kohn/
+
+
+## Themes
+
+The following themes emerged from multiple community groups reporting in with 
+similar experiences - whether positive or challenging - and areas of research to
+explore more in the future. 
+
+### Project Communication Strategy 
+  
+During periods of significant contributor growth, [community groups were
+reporting internally and externally regularly with the group's members, the 
+project at large, and at KubeCons. This made sense as we scaled ten thousand 
+casual and active contributors during these growth periods but it's now added up
+to duplicate meetings and too many update slides. The COVID-19 pandemic and an 
+increase in contributors outside of the North America Pacific Timezone made 
+regular meetings difficult. Chairs and other project leads asked for a more 
+streamline and consistent reporting and feedback mechanism. 
+    
+#### What We've Done
+
+- Changed [Community Meeting] cadence from weekly to monthly, and changed 
+meeting style from a "read out" of updates to more discussion oriented.
+- Encouraged groups to use asychronous methods for delivering updates and 
+gathering feedback. For example: Slack "standup" theads which feed into larger 
+scale reporting to a group on a mailing list.
+- Created an internal marketing group under SIG Contributor Experience to help
+  facilitate community communication.
+
+
+[Community Meeting]: https://k8s.dev/events/community-meeting/
+[community groups to report in and out]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
+[Governance requirements]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md#chair
+    
+#### Areas of Research 
+
+- Review SIG charters to re-focus goals of each group at the current state of
+  maturity
+- Expand on async meeting adoption and establish best practices 
+- Improving subproject communication and connective glue back to the sponsoring
+  SIG
+- Clarify conditions for archiving or putting a subproject in maintenance mode
+- Connecting with the end user community and understanding what they want to
+  hear from upstream
+  
+    
+### Refining the Contributor Lifecycle
+
+There are a number of ways to define membership to the Kubernetes project
+and its community groups. The project's community groups must define
+their own membership terms and then various other activities and roles.
+Membership can drive:
+
+- Kubernetes GitHub Organization membership 
+- Consensus 
+- Voting
+
+We define [membership] as contributors added to to one of the four GitHub 
```

Comment:
```suggestion
Steering Committee defines [membership] as contributors added to to one of the four Kubernetes GitHub 
```

Reply:
Maybe The project defines membership? This is generally how we define it in community-membership.md